downstream/modules/platform/proc-provision-ocp-storage-amazon-s3.adoc

```diff
@@ -0,0 +1,49 @@
+[id="provision-ocp-storage-amazon-s3_{context}"]
+
+
+= Configuring object storage on Amazon S3
+
+Red Hat supports Amazon Simple Storage Service (S3) for {HubName}.
+You can configure it at deployment on the `AutomationHub` custom resource (CR), or you can configure it for an existing deployment.
```

Comment:
@rooftopcellist 
I used "when deploying"; I can delete this if it would be better.
I changed "deployment" to "instance"; again, I'm happy to reword if there's a more appropriate word.